src/TritonRoute/src/db/obj/frBlock.h

```diff
@@ -289,7 +274,28 @@ class frBlock : public frBlockObject
     name2snet_[in->getName()] = in.get();
     snets_.push_back(std::move(in));
   }
-  void setBoundaries(const std::vector<frBoundary> in) { boundaries_ = in; }
+  const frBox& getDieBox() const {
+      return dieBox_;
+  }
```

Comment:
This looks equivalent to getBoundaryBBox but with a different name.  Let's pick one name.